src/backend/utils/sort/tuplesort_mk.c

```diff
@@ -3070,8 +3012,12 @@ tupsort_compare_datum(MKEntry *v1, MKEntry *v2, MKLvContext *lvctxt, MKContext *
 
 				return ((lvctxt->scanKey.sk_flags & SK_BT_DESC) != 0) ? -result : result;
 			}
+		case MKLV_TYPE_NONE:
```

Comment:
@leskin-in I have made a change in this part of code according to your [suggestion](https://github.com/greenplum-db/gpdb/pull/6927#discussion_r255458295).